src/main/java/hellos/Hellos.java

```diff
@@ -8,48 +8,27 @@ public class Hellos {
 
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
-
-    builder.append(andreasSaysHello());
-    builder.append(elySaysHello());
-    builder.append(jadenSaysHello());
```

Comment:
This is very definitely a Bad Thing. You appear to have removed _many_ of the other people's greetings, probably while handling merge conflicts. You definitely removed all three of these, and it looks like you removed Ken, Pat, and Tristan below.

You have to fix that before you can merge this in, and no one should approve this PR without confirming that these have been restored.